aicsimageio/readers/lif_reader.py

```diff
@@ -140,20 +140,22 @@ def _get_image_data(
         chunk: np.ndarray
             The image chunk as a numpy array.
         """
+        MISSING_DIM_SENTINAL_VALUE = -1
```

Comment:
Sorry I'm late in a response! For future cases like this or even cases where `-1` is a valid value, just a plain `object()` can be used as a truly unique sentinel (also I guess it's "sentinel" and not "sentinal", I definitely would have misspelled it as well 😅). E.g. `MISSING_DIM_SENTINAL_VALUE = object()`.

Reply:
yeah it's sentinel  with an e

Reply:
Object works in place of int for typing?? Fascinating.

Also lol.